### DISCUSSION TITLE
rx 6950 works?

### DISCUSSION THREAD

**Discussion by larini** [ChefKissInc/NootedRed#323] [Q&A]
Is the rx 6950 supported? How exactly install this kext?

**Comment by nisel11** [comment]
**Is the rx 6950 supported?**
RX 6950 XT supported, check [NootRX website](https://chefkissinc.github.io/applehax/nootrx/)

**How exactly install this kext?**
[Hackintosh guide](https://chefkissinc.github.io/guides/hackintosh/), or if you already have hackintosh on your PC then just copy kext to EFI/OC/Kexts folder and add kext to Config.plist

**This is the github of the NootedRed project which is responsible for supporting AMD Vega integrated graphics cards, for discrete graphics cards use [NootRX](https://github.com/ChefKissInc/NootRX)**
